package-lock.json

```diff
@@ -14239,6 +14245,247 @@
+					"integrity": "sha512-3GLWFAan2pbwBeoeNDoqGmSbrShORtgWfaWX0RJDivsUrpShh01saRM5RU/i4Zmf+whpBVEY5cA90Eq8Ub1N3w==",
+					"requires": {
+						"@babel/runtime": "^7.10.2",
+						"use-composed-ref": "^1.0.0",
+						"use-latest": "^1.0.0"
+					}
+				},
+				"react-use-gesture": {
+					"version": "7.0.16",
+					"resolved": "https://registry.npmjs.org/react-use-gesture/-/react-use-gesture-7.0.16.tgz",
+					"integrity": "sha512-gwgX+E+WQG0T1uFVl3z8j3ZwH3QQGIgVl7VtQEC2m0IscSs668sSps4Ss3CFp3Vns8xx0j9TVK4aBXH6+YrpEg=="
+				}
+			}
+		},
+		"@wp-g2/context": {
+			"version": "0.0.116",
+			"resolved": "https://registry.npmjs.org/@wp-g2/context/-/context-0.0.116.tgz",
+			"integrity": "sha512-tw21I/ATaGraiKJ2BS/lhA+pSJsFWLZz5L0s/M47xHcAfRcimNOBe6+W7Aud19rC5eCPiVCoLIdvzgdSvn/AoA==",
+			"requires": {
+				"@wp-g2/styles": "^0.0.116",
+				"@wp-g2/utils": "^0.0.116",
+				"lodash": "^4.17.19"
+			}
+		},
+		"@wp-g2/create-styles": {
+			"version": "0.0.116",
+			"resolved": "https://registry.npmjs.org/@wp-g2/create-styles/-/create-styles-0.0.116.tgz",
+			"integrity": "sha512-0s22CPPdsePS1AKgXqZ0Azuus2tBI5jQrRb50kEYwoFEjgMrUQqXZdqnrpuZvO2E7cvbSIrCEk9FPa++ydv37g==",
+			"requires": {
+				"@emotion/core": "^10.0.28",
+				"@emotion/is-prop-valid": "^0.8.8",
+				"@wordpress/is-shallow-equal": "^2.3.0",
+				"@wp-g2/substate": "^0.0.116",
+				"@wp-g2/utils": "^0.0.116",
+				"create-emotion": "^10.0.27",
+				"emotion": "^10.0.27",
+				"emotion-theming": "^10.0.27",
+				"lodash": "^4.17.19",
+				"mitt": "^2.1.0",
+				"rtlcss": "^2.6.1",
+				"styled-griddie": "^0.1.3"
+			},
+			"dependencies": {
+				"@choojs/findup": {
+					"version": "0.2.1",
+					"resolved": "https://registry.npmjs.org/@choojs/findup/-/findup-0.2.1.tgz",
+					"integrity": "sha512-YstAqNb0MCN8PjdLCDfRsBcGVRN41f3vgLvaI0IrIcBp4AqILRSS0DeWNGkicC+f/zRIPJLc+9RURVSepwvfBw==",
+					"requires": {
+						"commander": "^2.15.1"
+					}
+				},
+				"@emotion/is-prop-valid": {
+					"version": "0.8.8",
+					"resolved": "https://registry.npmjs.org/@emotion/is-prop-valid/-/is-prop-valid-0.8.8.tgz",
+					"integrity": "sha512-u5WtneEAr5IDG2Wv65yhunPSMLIpuKsbuOktRojfrEiEvRyC85LgPMZI63cr7NUqT8ZIGdSVg8ZKGxIug4lXcA==",
+					"requires": {
+						"@emotion/memoize": "0.7.4"
+					}
+				},
+				"@emotion/memoize": {
+					"version": "0.7.4",
+					"resolved": "https://registry.npmjs.org/@emotion/memoize/-/memoize-0.7.4.tgz",
+					"integrity": "sha512-Ja/Vfqe3HpuzRsG1oBtWTHk2PGZ7GR+2Vz5iYGelAw8dx32K0y7PjVuxK6z1nMpZOqAFsRUPCkK1YjJ56qJlgw=="
+				},
+				"chalk": {
+					"version": "2.4.2",
+					"resolved": "https://registry.npmjs.org/chalk/-/chalk-2.4.2.tgz",
+					"integrity": "sha512-Mti+f9lpJNcwF4tWV8/OrTTtF1gZi+f8FqlyAdouralcFWFQWF2+NgCHShjkCb+IFBLq9buZwE1xckQU4peSuQ==",
+					"requires": {
+						"ansi-styles": "^3.2.1",
+						"escape-string-regexp": "^1.0.5",
+						"supports-color": "^5.3.0"
+					}
+				},
+				"commander": {
```

Comment:
It's surprising to see `commander`, `postcss` and `rtlcss` on the list of dependencies. Is any of them used on the front-end?